src/index.js

```diff
@@ -475,22 +475,11 @@ function ncc (
       map = JSON.parse(map);
       // make source map sources relative to output
       map.sources = map.sources.map(source => {
-        // webpack:///webpack:/// happens too for some reason
-        while (source.startsWith('webpack:///'))
-          source = source.slice(11);
-        if (source.startsWith('//'))
-          source = source.slice(1);
-        if (source.startsWith('/'))
-          source = relative(process.cwd(), source).replace(/\\/g, '/');
-        if (source.startsWith('external '))
-          source = 'node:' + source.slice(9);
-        if (source.startsWith('./'))
-          source = source.slice(2);
-        if (source.startsWith('(webpack)'))
-          source = 'webpack' + source.slice(9);
-        if (source.startsWith('webpack/'))
-          return '/webpack/' + source.slice(8);
-        return sourceMapBasePrefix + source;
+        // webpack://[namespace]/[resourcePath]
```

Comment:
Looks like this no longer accounts for the `webpack:///webpack:///` case.

Reply:
Yes, look at [output.devtoolModuleFilenameTemplate](https://webpack.js.org/configuration/output/#outputdevtoolmodulefilenametemplate) , I think using webpack config is a better choice

Reply:
Thanks! Is there a way to add a test to confirm this fixes the issue?

Reply:
Sure! I've added tests